services/conference/src/scripts/models/TrackMiddleware/MuteLocal.ts

```diff
@@ -0,0 +1,26 @@
+import {reaction} from 'mobx'
+import {default as appLevel} from '@stores/AppLevel'
+import {connection} from '@models/api'
+import participants from '@stores/participants/Participants'
+
+reaction(() => appLevel.micOn,
+(micOn) => {
+  const tracks = connection.localTracks
+  for (const track of tracks) {
+    if (track.getType() === 'audio') {
+      micOn ? track.unmute() : track.mute()
+    }
+  }
+},
+)
+reaction(() => appLevel.cameraOn,

```

Comment:
no need to seperate `appLevel.cameraOn` and `participants.local.get().plugins.streamControl.muteVideo`
```
reaction(() => participants.local.get().plugins.streamControl.muteVideo, ...
```

Reply:
In this case, `micOn` and `cameraOn` in AppLevel is not needed anymore.

Reply:
I also considered about it. But when leave from one conference and join to another conference, is 'participants.local.get().plugins.streamControl.muteVideo' kept ?
If so, I'll remove appLevel store.